webdriver-spec.html

```diff
@@ -6030,7 +6030,7 @@
         </tr>
       </table>
 
-    <p>A <dfn>tick</dfn> is the basic unit of time over which actions
+    <p>A <dfn data-lt="tick|ticks">tick</dfn> is the basic unit of time over which actions
```

Comment:
It doesn’t care about upper- or lower case, but “remote end” in that list was probably added there before ReSpec supported taking the inner text implicitly. It’s safe to remove it, but that should probably be done in a separate commit.